.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "fluid-queue-setup"]
+	path = fluid-queue-setup
+	url = git@github.com:fluid-queue/fluid-queue-setup.git
```

Comment:
I would advise against using an SSH url here, because this submodule is only used read-only to build the docs.

If someone checks out the `fluid-docs` repository using HTTPS just to build the docs, then we would otherwise require them to have a GitHub SSH connection set-up.

Using a HTTPS url here allows anyone to just check out the docs with HTTPS to build the docs and if someone wants to make changes to the setup they can do so in their own fork anyways or can use the setup repository standalone and test their changes with the `spa` folder without having to have the docs checked out.

Reply:
Maybe we could use the HTTPS url on the `main` branch, and the SSH url for develop branches?